src/features/courses/course-route-validation.ts

```diff
@@ -102,13 +102,22 @@ export const updateCourseTopicValidation = {
 export const extendCourseTopicForUserValidation = {
     params: {},
     body: {
-        startDate: Joi.date().optional(),
-        endDate: Joi.date().optional(),
-        deadDate: Joi.date().optional()
+        extensions: Joi.object({
+            startDate: Joi.date().optional(),
+            endDate: Joi.date().optional(),
+            deadDate: Joi.date().optional()    
+        }).optional(),
+        studentTopicAssessmentOverride: Joi.object({
+            versionDelay: Joi.number(),
```

Comment:
Should these be explicitly optional?

Reply:
They should be, however it doesn't matter for now, I wouldn't hold up the PR for it, once it's merged I'll fix in my current branch